src/components/App.js

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import ButtonPanel from './ButtonPanel';
 import Display from './Display';
-import calculate from '../logic/calculate'; // eslint-disable-line
+import calculate from '../logic/calculate';
 
-const App = () => (
-  <>
-    <div className="row main-center cross-center h-100">
-      <div className="col">
-        <Display />
-        <ButtonPanel />
+const App = () => {
+  const [total, setTotal] = React.useState(null);
+  const [next, setNext] = React.useState(null);
+  const [operation, setOperation] = React.useState(null);
+  const handleClick = buttonName => {
+    const result = calculate({ total, next, operation }, buttonName);
+    setTotal(result.total);
+    setNext(result.next);
+    setOperation(result.operation);
```

Comment:
Great job showing your knowledge of Hooks 👍 

But for this project, you're supposed to use React Constructor method.;
Screenshot from [pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376019/) 👇 
![image](https://user-images.githubusercontent.com/31897434/111038686-a49a4a80-842a-11eb-91e8-9f8f369ba72f.png)


Keep in mind that the instructions on pathwright are bind with every other student to follow, and it would be great to also learn how Constructor works in React, I think that is the main reason you need to use it.